src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java

```diff
@@ -22,8 +22,8 @@
     private static final String FXML = "/view/TitleDescriptionDisplay.fxml";
 
     //FXML properties
-    private static final String USER_INFORMATION_WIDTH = "-fx-min-width: 95";
-    private static final String COMPANY_INTERNSHIP_INFORMATION_WIDTH = "-fx-min-width: 75";
+    private static final String PROFILE_INFORMATION_WIDTH = "-fx-min-width: 95";
+    private static final String COMPANY_APPLICATION_INFORMATION_WIDTH = "-fx-min-width: 75";
```

Comment:
Just to confirm, does company application refer to internship application i.e. our application class in this case?

Reply:
Thanks for the clarification Shawn, what I meant as this naming is, Company & Application to have the same Information Width. I will do the relevant changes to make it clearer.

Reply:
Ah I see! Thanks for the clarification Sean!